script.js

```diff
@@ -83,6 +83,7 @@ function visualControl($scope, $filter, $http) {
       r = {};
       r.startDate = Date.parse($filter('date')(d.startDate, 'dd/MMM/yyyy HH:mm:ss')) / 1000;
       r.endDate = Date.parse($filter('date')(d.endDate, 'dd/MMM/yyyy HH:mm:ss')) / 1000;
+      r.endDate += daySeconds; // Capture entire day
       if (r.startDate >= r.endDate) {
```

Comment:
If we want the dates to be inclusive of the end interval, could you change this line also to 

```
if (r.startDate > r.endDate) {
```

Thanks so much